client/src/pages/Admin/UsersIndex.jest.spec.js

```diff
@@ -31,7 +31,7 @@ const wrapperFactory = (mocks) => {
     //  plugins: [VueCompositionAPI],
     mount: {
       provide: apolloProvider,
-      type: "shallow",
+      type: "full",
```

Comment:
We'll want to refactor the selector on line 57 to use the `ref` attribute for making the selection of `q-item`s more precise